web/src/components/review-form.jsx

```diff
@@ -98,6 +101,54 @@ export default class ReviewForm extends React.Component {
     const offset = this.getOffset();
     const curSentences = this.props.sentences.slice(offset, offset + PAGE_SIZE);
 
+    if (mobileUI) {
+      let message = (<p>Swipe right to approve sentencee, swipe left to reject it.</p>);
+      if (this.state.page !== 0) {
+        message = (<p>You have succesfully reviewed your {this.state.page * PAGE_SIZE}th sentence!</p>)
```

Comment:
I think this should be an additional message, so the initial tutorial sentence does not get replaced